apps/services/license-api/src/app/modules/license/license.controller.ts

```diff
@@ -21,12 +22,17 @@ import {
 import { ApiHeader, ApiTags } from '@nestjs/swagger'
 import { LicenseId } from './license.types'
 import { NationalId } from '../../decorators/nationalId'
+import { IdsAuthGuard, Scopes } from '@island.is/auth-nest-tools'
+import { LicenseApiScope, getLicenseTypeScopes } from '@island.is/auth/scopes'
+import { LicenseTypeScopesGuard } from './guards/licenseTypeScope.guard'
 
 @ApiHeader({
   name: 'X-Param-NationalId',
   description: "The user's national id",
 })
 @Controller({ version: ['1'], path: 'users/.nationalId/licenses/' })
+@UseGuards(IdsAuthGuard, LicenseTypeScopesGuard)
+@Scopes(...getLicenseTypeScopes())
```

Comment:
LicenseTypeScopesGuard is quite specialised so I would suggest getting rid of the `@Scopes()` decorator and get the scope list directly.

Reply:
sounds good